### DISCUSSION TITLE
How to scale stage by its center

### DISCUSSION THREAD

**Discussion by ghost** [konvajs/konva#1687] [Q&A] [answered]
Hi,
I was wondering if there is any way to scale the stage by the center of the canvas, not the top left corner

**Comment by ghost** [comment]
I've already tried to use offset on the stage but with no luck, the scaling operation is always done by the corner

**Reply by ghost** [reply]
Here a reproduction example: https://codesandbox.io/p/sandbox/konva-click-bug-zfp9w6
@lavrton is this the expected behaviour?

**Comment by BASEFlyer** [comment, marked as answer]
https://konvajs.org/docs/sandbox/Zooming_Relative_To_Pointer.html

Use this as an example, just replace mouse position for center of canvas.

**Reply by ghost** [reply]
That's it!! Thank you!
